app/Controller/EventsController.php

```diff
@@ -2763,8 +2763,17 @@ public function freeTextImport($id) {
 		
 		if ($this->request->is('post')) {
 			App::uses('ComplexTypeTool', 'Tools');
+
+
+            Configure::write('debug', 2);
```

Comment:
:) already spoke w/ Andras about this, I'll create separate branches for separate issues. 
And thanks, have a nice evening and weekend as well!